README.md

```diff
@@ -42,6 +42,9 @@ in various programming languages.
 * **F\***: an ML-like effectful dependently typed functional programming language aimed at program verification  
   ([www](https://fstar-lang.org))
 
+* **Idris Effects**: library for algebraic effects and handlers in Idriss - inspired by Eff language  
+  ([github](https://github.com/idris-lang/Idris-dev/tree/master/libs/effects))  ([www](https://www.idris-lang.org/documentation/effects/))
```

Comment:
I think the "www" link is sufficient here: the GitHub link points to the middle of the source tree of a larger project, which isn't particularly enlightening.

Reply:
Done - github link removed: https://github.com/yallop/effects-bibliography/pull/56/commits/42a6d2520a7c1d865a360664bc2b150895a082d5